<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>434</width>
    <height>388</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0" colspan="3">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Output Folder</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="LEOutputFolder"/>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Input url</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="LEInputUrl"/>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="pBAddUrl">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="3">
       <widget class="QListWidget" name="listUrls"/>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="pBBrowseOutputFolder">
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="pBAddUrlsFromFile">
      <property name="text">
       <string>Add urls from file</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="pBDownload">
      <property name="text">
       <string>Download</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>434</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="mFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionFileQuit"/>
   </widget>
   <widget class="QMenu" name="mAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAboutAbout"/>
    <addaction name="actionAboutAboutQt"/>
    <addaction name="actionAboutHomeSite"/>
   </widget>
   <addaction name="mFile"/>
   <addaction name="mAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionFileQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAboutAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionAboutAboutQt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionAboutHomeSite">
   <property name="text">
    <string>Home site</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
